packages/tsconfig/package.json

```diff
@@ -7,8 +7,9 @@
     "type": "git",
     "url": "git+https://github.com/spotify/web-scripts.git"
   },
+  "main": "tsconfig.json",
   "files": [
-    "tsconfig.*.json"
+    "tsconfig*.json"
```

Comment:
Is the `*` necessary?

Reply:
Just based off an artless search, I think the `*` is the idiomatic way of adding specification to _which type of typescript configuration the file covers_.

![tsconfig](https://user-images.githubusercontent.com/5309910/71769726-b6ad5e80-2ef2-11ea-80ad-19d9ef30152f.png)

Reply:
@kirbysayshi is right though, we only export the one tsconfig.json now so this future-proofing move isn't necessary. Dropping it for the final PR.